packages/components/package.json

```diff
@@ -20,10 +20,11 @@
     "url": "git+https://github.com/chanzuckerberg/lp-design-system.git"
   },
   "scripts": {
-    "build": "npm run build:clean && npm run build:ts -- --sourceMap false",
+    "build": "npm run build:clean && npm run build:js",
     "build:clean": "rm -rf dist/",
-    "build:ts": "tsc -project . --rootDir src/ ",
-    "build:watch": "npm run build:ts -- --watch",
+    "build:js": "babel src --out-dir dist --extensions .js,.jsx,.ts,.tsx --ignore src/**/*.spec.tsx,src/**/*.stories.tsx",
```

Comment:
can you try `spec.*` and `stories.*` since we want to account for `spec.ts` as well. I'm pretty sure I've tried full regex or blob though and it didn't work, so it won't take `src/**/*.(spec|stories).(tsx?|jsx?)` or `src/**/*.{spec,stories}.{js,jsx,ts,tsx}` IIRC.

Reply:
hmm I'm trying
```
--ignore src/**/*.spec.*,src/**/*.stories.*
```
but that's not ignoring either type of file :/ 

just including one path `--ignore src/**/*.spec.*` works though

Reply:
I can get around it by including `--ignore` twice 😬

Reply:
I'll play around with it. Thanks testing it out 👍